R.swift/Generators/StringsGenerator.swift

```diff
@@ -0,0 +1,313 @@
+//
+//  StringsGenerator.swift
+//  R.swift
+//
+//  Created by Nolan Warner on 2016/02/23.
+//  Copyright © 2016 Nolan Warner. All rights reserved.
+//
+
+import Foundation
+
+struct StringsGenerator: Generator {
+  let externalStruct: Struct?
+  let internalStruct: Struct? = nil
+
+  init(localizableStrings: [LocalizableStrings]) {
+
+    let localized = localizableStrings.groupBy { $0.filename }
```

Comment:
The first `groupBy` collects files from different locales, these all share the same filename.

The `groupBySwiftNames` is used to check for duplicates and empty swift identifiers.
This is only done _per_ filename group.

This code should be extracted and done somewhere else in a generic method. Related:
- [Validation methods don't take duplicate identifiers into account](https://github.com/mac-cain13/R.swift/issues/175)
- [Add support for localised storyboards and nibs](https://github.com/mac-cain13/R.swift/issues/155)
- [Refactor warnings so that it's not an sideeffect anymore](https://github.com/mac-cain13/R.swift/issues/136)